packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -313,7 +316,7 @@ export default ({
     try {
       const data = yield call(api.createFiatDeposit, amount, id, currency)
       if (partner === 'YAPILY') {
-        const order = yield retry(100, 10000, AuthUrlCheck, data.paymentId)
+        const order = yield retry(30, 10000, AuthUrlCheck, data.paymentId)
```

Comment:
is this going to call API too often?

Reply:
It's 1 call every 10 seconds for 30 total retries. That's 5 minutes. From my testing Yapily can take a 3-5 minutes to process an order and there's reason to believe it could take longer in a production environment.